src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/formatter/LocationFormatter.kt

```diff
@@ -0,0 +1,15 @@
+package uk.gov.justice.digital.hmpps.welcometoprison.formatter
+
+import org.springframework.stereotype.Component
+
+@Component
+class LocationFormatter {
+
+  fun format(value: String?): String {
+    return when (value) {
+      "RECP" -> "Reception"
+      null -> ""
```

Comment:
think we'll probably want to throw an exception here - this is an unexpected case it would be better to be able to observe

Reply:
I checked prison-api and description can not be null so we can remove this condition as we always receive the value.